src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class ColorSupplier {

```

Comment:
The class should be declared as public to ensure it can be accessed from other classes outside of its package.